### PR TITLE
Added Navbar and Grid

### PR DESCRIPTION
**Pull request by Shivani-1524**
The components to be reviewed:
Two types of Navbars
50-50 grid layout, 70-30 grid layout and 3 column grid layouts

### CODE REVIEW

File: Components/Grids/grids.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  max-height: 100%;
+  align-items: center;
+  justify-content: center;
+}
+
+img {
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: contain;
+} */
```
Comment (rahulk31) [inline Components/Grids/grids.css]: Commented Code can be removed, if no longer required.

File: documentation.html [hunk 2]
```diff
@@ -360,6 +362,163 @@ <h2 class="rg-booky-title">Text and Icon Badge</h2>
...
+                            <div class="grid-2">
+                                <div class="img-wrapper">
+                                    <img src="./Assets/Images/avatar-img.jpg" alt="">
+                                </div>
+                                <div>
+                                    <p>GRID 2</p>
+                                </div>
+                            </div>
+                        </div>
+                    </div> -->
```
Comment (rahulk31) [inline documentation.html:386]: here as well

**Comment by iamalhera** [comment]
the first navbar get stucked in responsiveness, otherwise looks cool!, Go ahead to merge

**Comment by Shivani-1524** [comment]
> the first navbar get stucked in responsiveness, otherwise looks cool!, Go ahead to merge

Thanks for pointing it out!

**Pull request merged**